resolwe/flow/executors/docker/run.py

```diff
@@ -115,26 +146,26 @@ def start(self):
         group_file.file.flush()
         self.temporary_files.append(group_file)
 
-        mappings.append({'src': passwd_file.name, 'dest': '/etc/passwd', 'mode': 'ro,Z'})
-        mappings.append({'src': group_file.name, 'dest': '/etc/group', 'mode': 'ro,Z'})
+        volumes.append({'src': passwd_file.name, 'dest': '/etc/passwd', 'options': 'ro,Z'})
```

Comment:
It would be nice to make these options configurable.

Reply:
`self.volumes`? Or move `self.volumes = volumes.copy()` to the end of the function.

Reply:
`self.volumes` is no longer needed now that we got rid of hacky path translation, I forgot to remove it.